sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py

```diff
@@ -416,21 +447,29 @@ def set_reference_grasp(self):
         """
         Set the current grasp as a reference for interpolation
         """
-        self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions()
-        
+        self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions(
+        )
+
         if self.current_grasp.joints_and_positions is None:
-            #We try to activate ethercat hand again (detect if any controllers are running, and listen to them to extract the current position)
+            # We try to activate ethercat hand again (detect if any controllers
+            # are running, and listen to them to extract the current position)
             self.sr_lib.activate_etherCAT_hand()
-            #Some time to start receiving data
+            # Some time to start receiving data
             rospy.sleep(0.5)
-            self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions()
-        
+            self.current_grasp.joints_and_positions = self.sr_lib.read_all_current_positions(
+            )
```

Comment:
dammit lint :( is this better or worse than breaking at the = but needing a" \"? not sure...